virtcontainers/pkg/types/types.go

```diff
@@ -5,6 +5,18 @@
 
 package types
 
+// PciPath describes a PCI path which identifies a PCI device by where
+// it sits in the PCI topology.
+//
+// Has the format "xx/.../yy/zz" Here, zz is the slot of the device on
+// its PCI bridge, yy is the slot of the bridge on its parent bridge
+// and so forth until xx is the slot of the "most upstream" bridge on
+// the root bus.  If a device is connected directly to the root bus,
+// its PciPath is just "zz"
```

Comment:
Looks like, we are going to simply put the `DD` part of the device as the `PciPath` for cloud-hypervisor, right?

Reply:
That's right.